src/ducks/components/OneAppLoading.jsx

```diff
@@ -0,0 +1,134 @@
+            </div>
+            <div className="sto-app-header-content">
+              <h2 className="sto-app-header-title">
+                <Placeholder width="12rem" height="1.5rem" />
+              </h2>
+              <p className="sto-app-header-description">
+                <Placeholder width="30rem" height="1.25rem" />
+              </p>
+              <Placeholder
+                width="8rem"
+                height="2.5rem"
+                style={{ display: 'inline-block', 'margin-right': '0.5rem' }}
+              />
+              <Placeholder
+                width="8rem"
+                height="2.5rem"
+                style={{ display: 'inline-block' }}
+              />
+            </div>
+          </div>
+          <div className="sto-app-images">
+            <div className="sto-app-big-images">
+              <Placeholder width="10rem" height="8rem" />
+              <Placeholder width="10rem" height="8rem" />
+            </div>
+            <div className="sto-app-small-images">
+              <Placeholder width="5rem" height="4rem" />
+              <Placeholder width="5rem" height="4rem" />
+            </div>
+          </div>
+          <div className="sto-app-details">
+            <div className="sto-app-descriptions">
+              <div className="sto-app-description">
+                <h3 className="u-title-h3">Description</h3>
+                <div>
+                  <p>
+                    <Placeholder width={[8, 15]} height="1.25rem" />
+                  </p>
+                  <ul>
+                    <li>
+                      <Placeholder width={[15, 30]} height="1.25rem" />
+                    </li>
+                    <li>
+                      <Placeholder width={[15, 30]} height="1.25rem" />
+                    </li>
+                    <li>
+                      <Placeholder width={[15, 30]} height="1.25rem" />
+                    </li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+            <div className="sto-app-additional-details">
+              <h3 className="u-title-h3">Informations</h3>
+              <div className="sto-app-info">
+                <div className="sto-app-info-header">Categories</div>
+                <div className="sto-app-info-content">
+                  <Placeholder width={[6, 10]} height="1.25rem" />
+                </div>
+              </div>
+              <div className="sto-app-info">
+                <div className="sto-app-info-header">Version</div>
+                <div className="sto-app-info-content">
+                  <Placeholder width={[6, 10]} height="1.25rem" />
+                </div>
+              </div>
+              <div className="sto-app-info">
+                <div className="sto-app-info-header">Languages</div>
+                <div className="sto-app-info-content">
+                  <Placeholder width={[6, 10]} height="1.25rem" />
```

Comment:
Personal opinion: I am absolutely not convinced about passing directly style attributes in our components; we should either use classes or use styles, the most important thing should be code consistency.  What should be our guidlines on this point ? ping @ptbrowne @GoOz @Crash-- 

However, it's a task for later out of the scope of this PR, let's keep it as it for now.

Reply:
I tend to agree. It's bad for consistency and it opens the door for difficult maintenance.